MetaMask/metamask-extension
# chore: Add App Opened Metric Event

## **Description**
This PR adds tracking for when the MetaMask app is opened in order to track MAU/MTU.


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28927?quickstart=1)

## **Related issues**

Fixes:  https://github.com/MetaMask/MetaMask-planning/issues/3401

## **Manual testing steps**

1. Ensure MM is not "fullscreen" in any tabs
2. Open chrome dev tools for service worker 
3. Clear the network tab
4. Open metamask (via Dapp or by clicking on extension)
5. There should be one or more new network request for POST https://api.segment.io/v1/batch – One of those should should have event: "App Opened"

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
